spec/sunomono_test.rb

```diff
@@ -0,0 +1,366 @@
+      end
+
+      it 'Generate feature in en' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        feature_name = 'sunomono'
+
+        Dir.chdir(@project_name)
+
+        system "suno generate feature '#{feature_name}'", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(File.readlines("#{@project_name}/features/#{feature_name}.feature")).
+            to include("# language: en\n", "Feature: #{feature_name.capitalize} \n", "\n", "  Background:\n", "    # Insert steps\n", "    \n", "  Scenario: First Scenario\n", "    # Insert steps\n")
+        expect(File.readlines("#{@project_name}/features/step_definitions/#{feature_name}_steps.rb")).
+            to include("######### GIVEN #########\n", "\n", "######### WHEN #########\n", "\n", "######### THEN #########")
+      end
+
+      it 'Generates with alias command g' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        Dir.chdir(@project_name)
+
+        feature_name = 'rspec_sunomono'
+
+        system "suno g feature '#{feature_name}'", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(Dir.entries("#{@project_name}/features")).
+            to include("#{feature_name}.feature")
+        expect(Dir.entries("#{@project_name}/features/step_definitions")).
+            to include("#{feature_name}_steps.rb")
+        expect(Dir.entries("#{@project_name}/features/android/screens")).
+            to include("#{feature_name}_screen.rb")
+        expect(Dir.entries("#{@project_name}/features/ios/screens")).
+            to include("#{feature_name}_screen.rb")
+      end
+    end
+
+    context 'Try Generates a feature with a invalid argument' do
+      it 'Feature will be not created' do
+        system "sunomono new '#{@project_name}'", :out => File::NULL
+
+        feature_name = 'rspec_sunomono'
+
+        Dir.chdir(@project_name)
+
+        system "sunomono generate feature '#{feature_name}' invalid_argument", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(Dir.entries("#{@project_name}/features")).
+            not_to include("#{feature_name}.feature")
+        expect(Dir.entries("#{@project_name}/features/step_definitions")).
+            not_to include("#{feature_name}_steps.rb")
+        expect(Dir.entries("#{@project_name}/features/android/screens")).
+            not_to include("#{feature_name}_screen.rb")
+        expect(Dir.entries("#{@project_name}/features/ios/screens")).
+            not_to include("#{feature_name}_screen.rb")
+      end
+
+      it 'Feature will be not created using alias command suno with a invalid argument' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        feature_name = 'rspec_sunomono'
+
+        Dir.chdir(@project_name)
+
+        system "suno generate feature '#{feature_name}' invalid_argument", :out => File::NULL
+
+        Dir.chdir('../')
```

Comment:
You are already inside the project folder, why do you return to its parent?
With this, you need to add the `@project_name` in all `expect` sentences below.